### PR TITLE
Fix for kernel >=4.10

### PR DESCRIPTION
**Pull request by MikalaiDrabovich**
github.com/GPUOpen-Tools/CodeXL/issues/161

### CODE REVIEW

File: CodeXL/Components/PowerProfiling/Backend/AMDTPowerProfilingDrivers/inc/AMDTRawDataFileHeader.h [hunk 1]
```diff
@@ -83,7 +83,7 @@ typedef enum
     RAW_UNKNOWN_ERROR = 3,
 } RAW_Status;
 
-static const wchar_t g_RawFileMagicStr[] = L"_RAWHDR_";
+static const wchar_t g_RawFileMagicStr[] = {'', 'R', 'A', 'W', 'H', 'D', 'R', ''};
```
Comment (PolarNick239) [inline CodeXL/Components/PowerProfiling/Backend/AMDTPowerProfilingDrivers/inc/AMDTRawDataFileHeader.h]: Should be:

`static const wchar_t g_RawFileMagicStr[] = {'_', 'R', 'A', 'W', 'H', 'D', 'R', '_'};`

**Comment by PolarNick239** [comment]
What's the point in developing any application if it can't be run on the latest LTS release of the most popular Linux OS?

This bug was encountered by many users and it is still there for five months already!

You just need to merge this PR and upload new release. You even don't seems to do testing, so this will not take your time too! I know that I am looking a bit too emotional - sorry for that, but this triggers me a lot - CodeXL is the only way to profile OpenCL on AMD and its release can't be installed for five months!

And it looks like nobody from AMD cares! Thanks at least that this is open-source app, so that **nabar** and **MikalaiDrabovich** could contribute and make life of others easer - thanks them a lot!

**Comment by pwasiewi** [comment]
https://github.com/GPUOpen-Tools/CodeXL/issues/161#issuecomment-374645236

**Comment by aalok-agarwal** [comment]
Hi MikalaiDrabovich,
Thanks for analyzing the issue and providing pull request.
We had made the fixes similar to pull request #163 with some additional fixes via the commit id: 36c267d which is available on the latest repo. This fix is not available on CodeXL 2.5-25 release source code, as it was released prior to this commit.

You may apply the 36c267d patch to CodeXL 2.5-25 source code and build it.
Or, you may git-clone the latest CodeXL source (which already includes 36c267d) and build it.

Regards,
Aalok

**Pull request closed without merging**